griptape/drivers/web_search/duck_duck_go_web_search_driver.py

```diff
@@ -19,7 +19,9 @@ class DuckDuckGoWebSearchDriver(BaseWebSearchDriver):
 
     def search(self, query: str, **kwargs) -> ListArtifact:
         try:
-            results = self.client.text(query, region=f"{self.language}-{self.country}", max_results=self.results_count)
+            results = self.client.text(
+                query, region=f"{self.language}-{self.country}", max_results=self.results_count, **kwargs
+            )
```

Comment:
the plan here is to just leave it up to the user to figure out how to use the driver? not provide a few top-level fields that the drivers need to implement?